### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by calohmn**

### CODE REVIEW

File: site/content/architecture/component-view/Component-View.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ The diagram below provides an overview of the top level *logical* components.
 
 {{< figure src="../Top-Level.jpg" >}}
 
-The *MQTT* and *Rest Potocol Adapters* use the *Device Registry* to assert the registration status of *Devices* connecting to the adapters. The *Device Registry* issues a token asserting the status which the protocol adapters include when forwarding telemetry data and events to the *Hono Server* component. The *Hono Server* verifies the device registration status by means of validating the token and forwards the messages to the downstream *AMQP 1.0 Messaging Network* for delivery to consuming *Business Applications*.
+The *MQTT* and *Rest Protocol Adapters* use the *Device Registry* to assert the registration status of *Devices* connecting to the adapters. The *Device Registry* issues a token asserting the status which the protocol adapters include when forwarding telemetry data and events to the *Hono Server* component. The *Hono Server* verifies the device registration status by means of validating the token and forwards the messages to the downstream *AMQP 1.0 Messaging Network* for delivery to consuming *Business Applications*.
```
Comment (sophokles73) [inline site/content/architecture/component-view/Component-View.md]: There doesn't seem to be difference here. Any clue why this shows up in the diff view?

File: site/content/component/mqtt-adapter.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ The following table provides an overview of the configuration variables and corr
...
+| `HONO_REGISTRATION_HOST`<br>`--hono.registration.host` | yes | `localhost` | The IP address or name of the Device Registration service. The adapter uses this service to get an assertion regarding a device's registration status, i.e. whether it is enabled and if it is registered with a particular tenant. NB: This variable needs to be set to an address that can be resolved within the network the adapter runs on. When running as a Docker container, use Docker's `--network` command line option to attach the adapter container to the same network the Device Registration service container is running on. |
```
Comment (sophokles73) [inline site/content/component/mqtt-adapter.md]: There doesn't seem to be difference here. Any clue why this shows up in the diff view?

File: site/content/architecture/component-view/Component-View.md (hunk 1 above)
Comment (calohmn) [inline site/content/architecture/component-view/Component-View.md]: "Rest Potocol Adapters" -> "Rest P**r**otocol Adapters"

File: site/content/component/mqtt-adapter.md (hunk 2 above)
Comment (calohmn) [inline site/content/component/mqtt-adapter.md]: "veriable" -> "variable"

**Pull request merged**